npm/private/tar.bzl

```diff
@@ -3,7 +3,7 @@
 load("@aspect_bazel_lib//lib:repo_utils.bzl", "repo_utils")
 
 # TODO: use a hermetic tar from aspect_bazel_lib and remove this.
-def check_is_gnu_tar(rctx):
+def is_system_tar(rctx):
```

Comment:
`detect_system_tar` a better name for this since it returns the system tar type and not a boolean

Reply:
Yes that's better, thank you 👍